src/components/RouterComponent.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import { Login, RoomView, About } from ".";
+
+const RouterComponent = () => {
+  return (
+    <Routes>
+      {/*-------------------- home page---------------------*/}
+      <Route path="/" element={<Login />} />
+      {/*--------------------a page for room initialization---------------------*/}
+      <Route path="/room/:id" element={<RoomView />} />
+      {/*-------------------- a page for where the game actually takes place---------------------*/}
+      <Route path="/room/:id/playing" />
```

Comment:
i could be wrong but im not sure we need this? since the game will take place in the room/:id